src/components/Section/Package/Print.jsx

```diff
@@ -23,8 +25,59 @@ class Print extends SectionElement {
     this.sections = this.sections.bind(this)
     this.handlePrint = this.handlePrint.bind(this)
     this.state = {
-      printed: false
+      printed: false,
+      attachments: []
     }
+
+    this.supportsBlobs = false
+    try {
+      this.supportsBlobs = !!new window.Blob
+    } catch (e) {
+      this.supportsBlobs = false
+    }
+
+    this.getStoredAttachments = this.getStoredAttachments.bind(this)
+    this.displayAttachments = this.displayAttachments.bind(this)
+
+  }
+
+  download (id) {
+    if (!this.supportsBlobs) {
+      return
+    }
+
+    const attachment = this.state.attachments.find(x => x.id === id)
+    api.getAttachment(id).then((response) => {
+      const blob = blobFromBase64(response.data, 'application/octet-stream')
+      FileSaver.saveAs(blob, attachment.filename)
+    }).catch(() => {
+      this.setState({ errorMessage: i18n.t('application.attachments.upload.error.download') })
+    })
+  }
+
+  getStoredAttachments () {
+    api.listAttachments().then((response) => {
+      this.setState({ attachments: response.data || [], errorMessage: '' })
+    })
+  }
+
+  displayAttachments (items) {
+    return items.map((x, i) => {
+      return (
+        <tr key={`attachment-${x.id}`}>
+          <td>
+            <Show when={this.supportsBlobs}>
+              <a href="javascript:;;" aria-label={`Download ${x.filename}`} onClick={this.download.bind(this, x.id)}>
+                <strong>{`${i+1}. `}{x.description ? `${x.description} - ` : ''}</strong>{x.filename}
+              </a>
+            </Show>
+            <Show when={!this.supportsBlobs}>
+              <strong>{`${i+1}. `}{x.description ? `${x.description} - ` : ''}</strong>{x.filename}
+            </Show>
```

Comment:
This seems like a useful thing to extract to its own component.

Reply:
Yeah, this was definitely a copy-and-paste from the attachments page, which didn't get a design review or prior guidance either. Before we extract into a separate component, I'll have Ben take a look.

Reply:
@ryanhofdotgov what ther the design implications of this issue?

Reply:
@bpdesigns Before TrueTandem went off contract, they quickly put together the Attachments page which allow applicants to upload supporting documents. It didn't go through much of a design review and Jay had specifically flagged it as an area that needed attention. 

We've temporarily disabled that attachment feature for the usability study (and correspondingly all of our test sites) because of needing additional security mechanisms to support it, but not being able to add them because of limitations of our interim authority to test (IAT).